src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -0,0 +1,212 @@
+#include "OrderManager.h"
+#include "Drawing/SoftwareDrawingEngine.h"
+#include "GameState.h"
+#include "Graphics/ImageIds.h"
+#include "Input.h"
+#include "Localisation/Formatting.h"
+#include "Objects/CargoObject.h"
+#include "Objects/ObjectManager.h"
+#include "Ui/WindowManager.h"
+#include "Vehicle.h"
+#include "Vehicles/Orders.h"
+#include "Vehicles/VehicleManager.h"
+#include "World/StationManager.h"
+
+#include <sstream>
+
+namespace OpenLoco::Vehicles::OrderManager
+{
+    // TODO: Make this a fixed vector of size 63 (kMaxNumOrderPerVehicle) no need for it to be dynamic
+    std::vector<NumDisplayFrame> _displayFrames;
+    const std::vector<NumDisplayFrame>& displayFrames() { return _displayFrames; }
+
+    uint8_t* orders() { return getGameState().orders; }
+    uint32_t& numOrders() { return getGameState().numOrders; }
+
+    void sub_470795(const uint32_t removeOrderTableOffset, const int16_t sizeOfRemovedOrderTable)
```

Comment:
Any reason the static was removed here? It doesn't look like its used anywhere else.

Reply:
Ah, I was already using this function in another branch. Could also move it into the header in an upcoming PR, but I thought I might as well.

Reply:
I see, might also rename it then to something more sensible in this case.